modules/auxiliary/sqli/dlink/dlink_central_wifimanager_sqli.rb

```diff
@@ -0,0 +1,183 @@
+##
+# This module requires Metasploit: https://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+
+require 'csv'
+require 'digest'
+
+class MetasploitModule < Msf::Auxiliary
+  include Msf::Exploit::Remote::HttpClient
+  include Msf::Exploit::SQLi
+
+  def initialize(info = {})
+    super(
+      update_info(
+        info,
+        'Name' => 'D-Link Central WiFiManager SQL injection',
+        'Description' => %q{
+          This module exploits a SQLi vulnerability found in
+          D-Link Central WiFi Manager CWM(100) before v1.03R0100_BETA6. The
+          vulnerability is an exposed API endpoint that allows the execution
+          of SQL queries without authentication, using this vulnerability, it's
+          possible to retrieve usernames and password hashes of registered users,
+          device configuration, and other data, it's also possible to add users,
+          or edit database informations.
+        },
+        'License' => MSF_LICENSE,
+        'Author' =>
+          [
+            'M3@ZionLab from DBAppSecurity',
+            'Redouane NIBOUCHA <rniboucha[at]yahoo.fr>' # Metasploit module
+          ],
+        'References' => [
+          ['CVE', '2019-13373'],
+          ['URL', 'https://unh3x.github.io/2019/02/21/D-link-(CWM-100)-Multiple-Vulnerabilities/']
+        ],
```

Comment:
Just a consistency request, the lists for `Author` and `Actions` here are formatted differently. The reason rubocop did not warn is that the `[` is inline reducing the indentation level expectations.
```suggestion
        'References' => 
          [
            ['CVE', '2019-13373'],
            ['URL', 'https://unh3x.github.io/2019/02/21/D-link-(CWM-100)-Multiple-Vulnerabilities/']
          ],
```